tests/tests-tax.php

```diff
@@ -179,6 +249,25 @@ public function test_display_tax_rates() {
 		$this->assertFalse( edd_display_tax_rate() );
 	}
 
+	public function test_cart_needs_tax_address_fields() {
+		$this->assertInternalType( 'bool', edd_cart_needs_tax_address_fields() );
+		$this->assertTrue( edd_cart_needs_tax_address_fields() );
+	}
+
+	public function test_cart_needs_tax_address_fields_false() {
+
+		// Prepare test
+		global $edd_options;
```

Comment:
Can you change this to use edd_get_option instead of the global variable please?